<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Images_QW</class>
 <widget class="QWidget" name="Images_QW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>4Safety</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="Images_QGL">
     <item row="2" column="0">
      <layout class="QGridLayout" name="Gallery_QGL">
       <property name="spacing">
        <number>10</number>
       </property>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="logo_QL">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Banco de imagens</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="close_QPB">
     <property name="text">
      <string>Fechar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="train_QPB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Adicionar ao banco de dados</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="delete_image_QPB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Deletar selecionada</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>delete_image_QPB</tabstop>
  <tabstop>train_QPB</tabstop>
  <tabstop>close_QPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
